src/app/pages/stats-page/stats-page.component.html

```diff
@@ -31,15 +41,46 @@
       <div class="break"></div>
       <div class="big-number">
         <div class="big-number-value">{{ info.minutes_to_win | minutes }}</div>
-        <div class="big-number-title">Estimated Time To Win</div>
+        <div class="big-number-title">
+          Estimated Time To Win
+          <div class="infoIcon" [popper]="tooltipTimeToWin"
+               [popperTrigger]="'hover'"
```

Comment:
1. You can just do `popperTrigger="hover"` in such cases.
2. You can define default settings, such as trigger, etc in `NgxPopperModule.forRoot({})`.